.testcaferc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
what made you choose e2e tests over unit tests?

Reply:
Hey André. Nice to meet you and thanks for the feedback
I didn't rely on E2E for the testing safety net. That was only a last-minute experiment since I wanted to learn Testcafe. Most of the logic is on the Kotlin side which has its own tests (most done with TDD).

Reply:
if you're referring to GUI unit tests, I could also have done them. But given the effort to create and maintain them (and given the very basic UI), why not apply that effort at a full-stack test? Besides manual testing, that's the only way to really guarantee the functionality is properly working. If the GUI gets more complex, we can also bet on those.